distributor/cmd/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang:1.19-alpine AS builder
+
+ARG GOFLAGS=""
+ENV GOFLAGS=$GOFLAGS
+ENV GO111MODULE=on
```

Comment:
`GO111MODULE` is on by default since Go 1.16.

Reply:
Good call, now removed.